internal/format/markdown_table.go

```diff
@@ -55,7 +55,7 @@ const (
 			| Name | Version |
 			|------|---------|
 			{{- range .Module.Requirements }}
-				| {{ name .Name }} | {{ tostring .Version | default "n/a" }} |
+				| <a name="requirements_{{ .Name }}"></a> [{{ name .Name }}](#requirements_{{ .Name }}) | {{ tostring .Version | default "n/a" }} |
```

Comment:
This is going to be a very (very) repetitive change if you want to do it this way. I'd suggest create new template function (like `anchorName` or something) and move the `<a name=....></a>...` into that. So effectively the change will become:

```diff
-				| {{ name .Name }} | {{ tostring .Version | default "n/a" }} |
+				| {{ anchorName .Name }} | {{ tostring .Version | default "n/a" }} |
```

Reply:
That's a good point. I'll add a new function for this.

Reply:
Finished. Thanks for the comment!